### DISCUSSION TITLE
Error: 没有找到页面实例。

### DISCUSSION THREAD

**Discussion by Xu213** [NervJS/taro#12621] [Q & A]
将app.ts改为app.tsx后找不到入口文件
![image](https://user-images.githubusercontent.com/48984582/194699714-473086a1-6235-4eca-bb89-893e8858116a.png)

**Comment by TheKonka** [comment]
https://github.com/NervJS/taro/issues/12560#issuecomment-1256875897

我猜测是 ```<Theme />``` 这个组件里面 条件渲染了 `children`
